### PR TITLE
OCPBUGS-17096: [release-4.12] Cleanup CRBs for precaching/backup (backport from #626)

### PR DESCRIPTION
**Pull request by jc-rh**
* Cleanup CRBs for precaching/backup

* Update 00-assert.yaml

---------

### CODE REVIEW

File: tests/kuttl/tests/backup-fail/05-errors.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
+#MCVs
+apiVersion: view.open-cluster-management.io/v1beta1
+kind: ManagedClusterView
```
Comment (irinamihai) [inline tests/kuttl/tests/backup-fail/05-errors.yaml:3]: Hmm, are these MCVs (fir spokes 1, 2, 5, 6) duplicated below?

Comment (jc-rh) [reply tests/kuttl/tests/backup-fail/05-errors.yaml:3]: Updated

**Pull request merged**